ansible_deployer/command_line.py

```diff
@@ -107,20 +107,25 @@ def create_workdir(timestamp: str):
     #
     #TODO: Add locking of the directory
 
-    if short_ts not in os.listdir(conf["global_paths"]["work_dir"]):
-        seq_path = os.path.join(date_dir, f"{conf['file_naming']['sequence_prefix']}0000")
-    else:
-        sequence_list = os.listdir(date_dir)
-        sequence_list.sort()
-        new_sequence = int(sequence_list[-1].split(conf['file_naming']['sequence_prefix'])[1]) + 1
-        seq_path = os.path.join(date_dir, f"{conf['file_naming']['sequence_prefix']}"
-                                          f"{new_sequence:04d}")
+    try:
+        if short_ts not in os.listdir(conf["global_paths"]["work_dir"]):
+            seq_path = os.path.join(date_dir, f"{conf['file_naming']['sequence_prefix']}0000")
+        else:
+            sequence_list = os.listdir(date_dir)
+            sequence_list.sort()
+            new_sequence = int(sequence_list[-1].split(conf['file_naming']['sequence_prefix'])[1])\
+            + 1
+            seq_path = os.path.join(date_dir, f"{conf['file_naming']['sequence_prefix']}"
+                                              f"{new_sequence:04d}")
+    except Exception as exc:
+        logger.critical("Failed to list work dir due to: %s", exc)
```

Comment:
We should add tests for those criticals. Maybe a second configuration file with a different workdir run from another .sh ?